Uzduotis06/Program.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+
+namespace uzduotis_6
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            /*
+                       5  Išveskite pasirinkto skaičiaus daugybos lentelę. 
+                            Pvz.: 
+                            5 * 0 = 0
+                            5 * 1 = 5
+                                …
+                            5 * 10 = 50
+
+                       6   Susikurkite sveikojo skaičiaus kintamąjį su dviženklio skaičiaus reikšme. Išveskite į ekraną šio skaičiaus skaitmenų sandaugą.
+
+                        */
+
+
+            // 5 uzduotis 
+
+            int skaicius_a = 7;
+            int skaicius_b = 0;
+
+            int rezultatas = skaicius_a * skaicius_b; //skaiciavimas
+            Console.WriteLine($"{skaicius_a} * {skaicius_b} = {rezultatas}"); // isvestis
+            int rezultatas1 = skaicius_a * ++skaicius_b; //skaiciavimas
+            Console.WriteLine($"{skaicius_a} * {skaicius_b} = {rezultatas1}"); // isvestis
+            int rezultatas2 = skaicius_a * ++skaicius_b; //skaiciavimas
+            Console.WriteLine($"{skaicius_a} * {skaicius_b} = {rezultatas2}"); // isvestis
+            int rezultatas3 = skaicius_a * ++skaicius_b; //skaiciavimas
+            Console.WriteLine($"{skaicius_a} * {skaicius_b} = {rezultatas3}"); // isvestis
+            int rezultatas4 = skaicius_a * ++skaicius_b; //skaiciavimas
+            Console.WriteLine($"{skaicius_a} * {skaicius_b} = {rezultatas4}"); // isvestis
+            int rezultatas5 = skaicius_a * ++skaicius_b; //skaiciavimas
+            Console.WriteLine($"{skaicius_a} * {skaicius_b} = {rezultatas5}"); // isvestis
+            int rezultatas6 = skaicius_a * ++skaicius_b; //skaiciavimas
+            Console.WriteLine($"{skaicius_a} * {skaicius_b} = {rezultatas6}"); // isvestis
+            int rezultatas7 = skaicius_a * ++skaicius_b; //skaiciavimas
+            Console.WriteLine($"{skaicius_a} * {skaicius_b} = {rezultatas7}"); // isvestis
+            int rezultatas8 = skaicius_a * ++skaicius_b; //skaiciavimas
+            Console.WriteLine($"{skaicius_a} * {skaicius_b} = {rezultatas8}"); // isvestis
+            int rezultatas9 = skaicius_a * ++skaicius_b; //skaiciavimas
+            Console.WriteLine($"{skaicius_a} * {skaicius_b} = {rezultatas9}"); // isvestis
+            int rezultatas10 = skaicius_a * ++skaicius_b; //skaiciavimas
+            Console.WriteLine($"{skaicius_a} * {skaicius_b} = {rezultatas10}"); // isvestis
+
```

Comment:
Taip parašytą kodą sunku skaityti, be to turite daug kintamųjų (`rezultatas1`, ..., `rezultatas10`) kurie realiai nėra reikalingi. Galite skaičiavimą atlikti tiesiog išvedime:
`Console.WriteLine($"{skaicius_a} * {skaicius_b} = {skaicius_a * skaicius_b++}");`
Kaip matote iš mano pavyzdžio `++skaicius_b` pakeičiau į `skaicius_b++` nes taip atrodo logiškiau. Pirmiausia viską atspausdiname ir galiausiai padidiname skaičių prieš kitą spausdinimą.